buf/registry/policy/v1beta1/upload_service.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.registry.policy.v1beta1;
+
+import "buf/registry/policy/v1beta1/configuration.proto";
+import "buf/registry/policy/v1beta1/policy.proto";
+import "buf/registry/policy/v1beta1/version.proto";
+import "buf/registry/priv/extension/v1beta1/extension.proto";
+import "buf/validate/validate.proto";
+
+option go_package = "buf.build/gen/go/bufbuild/registry/protocolbuffers/go/buf/registry/policy/v1beta1";
+
+// Upload contents.
+service UploadService {
+  // Upload contents for given set of Policies.
+  //
+  // Contents are expected to be a Configuration.
+  rpc Upload(UploadRequest) returns (UploadResponse) {}
+}
+
+message UploadRequest {
+  option (buf.registry.priv.extension.v1beta1.message).request_only = true;
+
+  // Content to upload.
+  message Content {
+    // The Policy of the reference.
+    PolicyRef policy_ref = 1 [(buf.validate.field).required = true];
+    // The Configuration of the Content.
+    Configuration configuration = 2 [(buf.validate.field).required = true];
```

Comment:
Currently we are uploading just the `Configuration` structure. This would mean that if there was a comment change in the policy definition file but nothing else, for example, then no new `Version` is created. We would also not consider the policy configuration as "content" and would have no knowledge of the original source, just the `Configuration` shape.